tox.ini

```diff
@@ -92,6 +92,7 @@ skip_install = true
 changedir = doc/scapy
 deps = sphinx>=2.4.2
        sphinx_rtd_theme
+       python-can
```

Comment:
All good to me except this. We can't allow dependencies in `docs` because downstream maintainers need to be able to build it too without using tox.

Reply:
ok, I'll check how I can fix the build of api docs without python-can